apps/www/pages/politikfragebogen/[id].js

```diff
@@ -0,0 +1,61 @@
+import { createGetServerSideProps } from '../../lib/apollo/helpers'
+import Page from '../../components/PoliticsQuestionnaire/Person'
+import fs from 'node:fs/promises'
+import path from 'node:path'
+import { csvParse } from 'd3'
+
+import {
+  QUESTION_TYPES,
+  leftJoin,
+} from '../../../www/components/PoliticsQuestionnaire/config'
+
+export default ({ responses, authorData }) => (
+  <Page responses={responses} authorData={authorData} />
+)
+
+export const getServerSideProps = createGetServerSideProps(
+  async ({
+    ctx: {
+      params: { id },
+    },
+  }) => {
+    const data = await fs.readFile(
+      path.join(
+        process.cwd(),
+        'public/static/politicsquestionnaire2023/submissions_dummy_data.csv',
+      ),
+      'utf-8',
+    )
+    const responses = csvParse(data)
+
+    const responsesWithTypes = leftJoin(
+      responses,
+      QUESTION_TYPES,
+      'questionSlug',
+    )
+
+    const responsesWithTypesById = responsesWithTypes.filter(
+      (d) => d.uuid === id,
+    )
+
+    const questionAnswerPair = responsesWithTypesById.map((d) => {
+      return {
+        type: d.type,
+        question: d.question,
+        answer: d.answer,
+        options: d.options || [],
+      }
+    })
+
+    const authorData = responsesWithTypesById.map((d) => {
+      return { id: d.uuid, name: d.name }
+    })
```

Comment:
naive question: wouldn't it make more sense to do all this preprocessing once and for all outside of the app (with a script or something) and instead of the "raw" csv data, store a processed json in `public/static`?
disclaimer: maybe that's not the best idea but i don't really the benefit of processing the data here.